worker/utils/utils.js

```diff
@@ -49,6 +51,27 @@ module.exports = {
     })
   },
 
+  async encryptJob(string1, string2){
+    const secret = this.retrievePassword() + string1 + string2
+    const salt = this.generateSalt();
+    const digest = crypto.scryptSync(secret, salt, 64);
+    return digest.toString('hex');
+  },
+
+  async decryptJob(digest, string1, string2){
```

Comment:
You're not decrypting: you're checking a password. Additionally, this can be defined in terms of `pwhashJob`, right?